### PR TITLE
Resize NumFOCUS Logo

### PR DESCRIPTION
**Pull request by RMeli**
Resized NumFOCUS logo.

**Please ignore this PR if you don't feel the NumFOCUS logo on the current `README.md` is too big.**

### CODE REVIEW

**Comment by IAlibay** [comment]
For context, here is the fiscal-sponsor template: https://github.com/numfocus/templates/blob/master/fiscal-sponsor-readme-attribution.md

File: README.rst [hunk 1]
```diff
@@ -16,12 +16,11 @@ MDAnalysis also includes widely used analysis algorithms in the `MDAnalysis.anal
...
 
-.. image:: https://raw.githubusercontent.com/numfocus/templates/master/images/numfocus-logo.png
-  :height: 60px
-  :target: https://numfocus.org/project/mdanalysis
-  :align: center
-  :alt: NumFOCUS
-  
+.. raw:: html
+
+   <img src="https://raw.githubusercontent.com/numfocus/templates/master/images/numfocus-logo.png" width="25%" alt="NumFOCUS">
```
Comment (IAlibay) [inline README.rst:21]: I see in the commit history that you tried several things, any reason (beyond flair) why centering was ditched here?

The template specifically refers to a centered logo, I don't know how much they care here though.

Comment (RMeli) [reply README.rst:21]: None worked... =(

Comment (RMeli) [reply README.rst]: Ok, the official template works as expected.

File: README.rst [hunk 2]
```diff
@@ -16,12 +16,17 @@ MDAnalysis also includes widely used analysis algorithms in the `MDAnalysis.anal
...
-.. image:: https://raw.githubusercontent.com/numfocus/templates/master/images/numfocus-logo.png
-  :height: 60px
-  :target: https://numfocus.org/project/mdanalysis
-  :align: center
-  :alt: NumFOCUS
-  
+.. raw:: html
+
+   <div align="center">
+      <a href="<PROJECT_NUMFOCUS_SPONSORSHIP_LINK>">
```
Comment (IAlibay) [inline README.rst:22]: The link here should target `https://numfocus.org/project/mdanalysis`?

File: README.rst [hunk 3]
```diff
@@ -16,12 +16,17 @@ MDAnalysis also includes widely used analysis algorithms in the `MDAnalysis.anal
...
+         <img height="60px" 
+            src="https://raw.githubusercontent.com/numfocus/templates/master/images/numfocus-logo.png" 
+            align="center">
+      </a>
+   </div>
+
+
```
Comment (IAlibay) [inline README.rst:29]: (feel free to ignore) In the end result this feel a little bit crowded. At least style-wise, I'd vote for a newline between the logo and the next line?

File: README.rst [hunk 4]
```diff
@@ -16,12 +16,18 @@ MDAnalysis also includes widely used analysis algorithms in the `MDAnalysis.anal
...
-  :align: center
-  :alt: NumFOCUS
-  
+.. raw:: html
+
+   <div align="center">
+      <a href="https://numfocus.org/project/mdanalysis">
+         <img height="60px" 
+            src="https://raw.githubusercontent.com/numfocus/templates/master/images/numfocus-logo.png" 
+            align="center">
```
Comment (lilyminium) [inline README.rst:25]:
```suggestion
            align="center" alt="NumFOCUS logo">
```

**Pull request merged**